packages/e2e/cypress/e2e/mapOverlays.spec.js

```diff
@@ -35,7 +35,10 @@ const urlToRouteRegex = url => {
     throw new Error(`'${url}' is not a valid report url: it must contain a 'report' query param`);
   }
 
-  return new RegExp(`measureData\\?(.*&|)mapOverlayCode=${mapOverlayCode}(&|$)`);
+  // Allows responses for both versions of the frontend
+  return new RegExp(
+    `(measureData\\?(.*&|)mapOverlayCode=${mapOverlayCode}(&|$)|legacyMapOverlayReport/${mapOverlayCode})`,
```

Comment:
This is matching against both versions of the frontend, it means we can actually run an e2e test between the two versions which is really only happening in this one release. We could delete the first half of this regex after this release